source/assets/css/sass.css.scss

```diff
@@ -6,7 +6,8 @@
   'vendor/normalize',
   'vendor/sanitize',
   'vendor/html5-boilerplate',
-  'vendor/syntax',
+  // 'vendor/syntax',
```

Comment:
we can just remove this

Reply:
Sorry to reopen this convo, but it looks like git/hub lost the changes from [this commit](https://github.com/sass/sass-site/pull/421/commits/be4d0d17ee3888b9747662aaf7739fb0fa37e088).